frontend/src/pages/agreements/approve/ApproveAgreement.hooks.js

```diff
@@ -177,6 +190,56 @@ const useApproveAgreement = () => {
         budgetLinesToExecutingMessages
     ]);
 
+    /**
+     * @param {Object[]} originalBudgetLines - The original budget lines
+     * @param {Object[]} cans - The CAN data retrieved from the RTL Query
+     * @returns {Object[]} The updated budget lines
+     */
+    function createUpdatedBudgetLines(originalBudgetLines, cans) {
```

Comment:
🤔 not sure what to call this function but it takes in budgetlines, parses the change requests, and returns the budgetlines as if they were approved